### PR TITLE
Feature/reg wait role

### LINKED ISSUE #95: Move wait+register host into role
Following snippet is repeated in all scenarios:
```tasks:
    - name: Wait for host
      script: wait_for_public_ip.py "{{ lookup('env', 'BASTION_PUBLIC_IP') }}" 120
      args:
        executable: python3
    - name: Register host
      add_host:
        hostname: server
        ansible_ssh_host: "{{ lookup('env', 'BASTION_PUBLIC_IP') }}"
        ansible_ssh_user: "linux"
        groups: scn4_ecs```

Maybe in make sense to move this to a role

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Closes #95

### CODE REVIEW

File: playbooks/roles/register_host/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Prepare inventory
+  become: yes
```
Comment (outcatcher) [inline playbooks/roles/register_host/tasks/main.yml]: There is `become: yes` here and in each playbook — is it required?

File: playbooks/roles/register_host/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+dnsname_or_ip: "{{ hostvars.bastion.ansible_host }}"
+hostname: "scenario_host"
```
Comment (outcatcher) [inline playbooks/roles/register_host/defaults/main.yml]: Maybe should be `server`, or should not be overridden in each playbook?

File: playbooks/roles/register_host/tasks/main.yml (hunk 1 above)
Comment (anton-sidelnikov) [reply playbooks/roles/register_host/tasks/main.yml]: yep, not required

File: playbooks/roles/register_host/defaults/main.yml (hunk 2 above)
Comment (anton-sidelnikov) [reply playbooks/roles/register_host/defaults/main.yml]: ok

File: playbooks/scenario1_setup.yml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 - name: Register existing settings
   hosts: local
-  tasks:
-    - name: Wait for host
-      script: wait_for_public_ip.py {{ hostvars.bastion.ansible_host }} 120
-      args:
-        executable: python3
+  roles:
+  - register_host
+  vars:
```
Comment (outcatcher) [inline playbooks/scenario1_setup.yml]: `vars` seems to be relevant only for `register_host` role

File: playbooks/scenario2_server_stop.yml [hunk 4]
```diff
@@ -1,12 +1,11 @@
...
   hosts: local
-  tasks:
-   - name: Register host
-     add_host:
-       hostname: "{{ lookup('env', 'SERVER_PUBLIC_IP') }}"
-       ansible_ssh_user: "linux"
-       groups: scn2_ecs
+  roles:
+  - register_host
+  vars:
```
Comment (outcatcher) [inline playbooks/scenario2_server_stop.yml]: Same as in previous comment

File: playbooks/scenario2_setup.yml [hunk 5]
```diff
@@ -1,16 +1,11 @@
...
-        executable: python3
-    - name: Register host
-      add_host:
-        hostname: web_server
-        ansible_ssh_host: "{{ lookup('env', 'SERVER_PUBLIC_IP') }}"
-        ansible_ssh_user: "linux"
-        groups: scn2_ecs
+  roles:
+  - register_host
+  vars:
```
Comment (outcatcher) [inline playbooks/scenario2_setup.yml]: Same as in previous comment

File: playbooks/scenario3_setup.yml [hunk 6]
```diff
@@ -1,17 +1,11 @@
...
-        executable: python3
-    - name: Register host
-      add_host:
-        hostname: server
-        ansible_ssh_host: "{{ lookup('env', 'SERVER_PUBLIC_IP') }}"
-        ansible_ssh_user: "linux"
-        groups: scn3_ecs
+  roles:
+  - register_host
+  vars:
```
Comment (outcatcher) [inline playbooks/scenario3_setup.yml]: Same as in previous comment

File: playbooks/scenario4_setup.yml [hunk 7]
```diff
@@ -1,17 +1,11 @@
...
-        executable: python3
-    - name: Register host
-      add_host:
-        hostname: server
-        ansible_ssh_host: "{{ lookup('env', 'BASTION_PUBLIC_IP') }}"
-        ansible_ssh_user: "linux"
-        groups: scn4_ecs
+  roles:
+  - register_host
+  vars:
```
Comment (outcatcher) [inline playbooks/scenario4_setup.yml]: Same as in previous comment

**Pull request merged**